embed/embed.go

```diff
@@ -315,7 +325,15 @@ func (f *_escFile) Close() error {
 }
 
 func (f *_escFile) Readdir(count int) ([]os.FileInfo, error) {
-	return nil, nil
+	var files []os.FileInfo
+	if f.isDir{
+		for _,filename := range f.childs {
+			fileHandle, _ := FS(false).Open("/"+f.local+"/"+filename)
+			stat, _ := fileHandle.Stat()
```

Comment:
A `_escFile` already implements os.FileInfo, so we shouldn't have to call .Stat() here.